libs/remix-ui/debugger-ui/src/lib/vm-debugger/dropdown-panel.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useRef, useEffect } from 'react'
+import { TreeView, TreeViewItem } from '@remix-ui/tree-view'
```

Comment:
An error is thrown while unfolding an item from the treeview. 
Also the way it is formmatted doesn't seem to be accurate
![Screenshot from 2020-10-08 11-47-00](https://user-images.githubusercontent.com/6940742/95442686-fe6a8600-095b-11eb-8aa4-fb89ebcbdf44.png)

Reply:
^ that seems to be fixed